Integrations/Home Assistant/heishamon.yaml

```diff
@@ -615,7 +633,18 @@ sensor:
           {%- else -%}
             n/a
           {%- endif -%}
-
+  #Calculation Energy consumed (kWh) - DHW
+  - platform: integration
+    source: sensor.aquarea_dhw_power_consumed
+    name: aquarea_dhw_power_consumed_kwh
```

Comment:
Could you set nice human readable names for these? So something like "Aquarea DHW Power Consumed kWh" instead?

Reply:
done